<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogGridCenterSelect</class>
 <widget class="QDialog" name="dialogGridCenterSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Grid Center Position</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_BL">
       <property name="text">
        <string>Bottom Left</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_C">
       <property name="text">
        <string>Center</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButton_B">
       <property name="text">
        <string>Bottom</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_TL">
       <property name="text">
        <string>Top Left</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_L">
       <property name="text">
        <string>Left</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_T">
       <property name="text">
        <string>Top</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="radioButton_BR">
       <property name="text">
        <string>Bottom Right</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="radioButton_R">
       <property name="text">
        <string>Right</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButton_TR">
       <property name="text">
        <string>Top Right</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">centerSelectBtnGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialogGridCenterSelect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogGridCenterSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="centerSelectBtnGroup"/>
 </buttongroups>
</ui>
